docs/topics/api/reviews.rst

```diff
@@ -114,6 +115,7 @@ If successful a :ref:`review object <review-detail-object>` is returned.
     :<json string|null body: The text of the review.
     :<json string|null title: The title of the review.
     :<json int rating: The rating the user wants to give as part of the review.
+    :<json int version: The add-on version id the review applies to.
```

Comment:
Oh damn, I wanted to back out that change. I'll remove this line again, that was a false alarm on my side. Sorry about that.